[8][pyyszhogzu] 로그인 시작
[8][pyyszhogzu] 로그인 종료
[8][pyyszhogzu] 로그인 검증 시작
[8][pyyszhogzu] 동의여부 체크 시작
[8][pyyszhogzu] Traceback (most recent call last):
  File "/instahelp_web/common/fun.py", line 894, in login
    if agree_check(account_id, tab_index, driver, wait):
  File "/instahelp_web/common/fun.py", line 803, in agree_check
    if agree.text == 'Instagram 앱을 사용하려면 다음 항목에 동의해야 합니다':
  File "/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/webelement.py", line 90, in text
    return self._execute(Command.GET_ELEMENT_TEXT)["value"]
  File "/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/webelement.py", line 395, in _execute
    return self._parent.execute(command, params)
  File "/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame
  (Session info: chrome-headless-shell=126.0.6478.61); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
Stacktrace:
#0 0x55b2a76506aa <unknown>
#1 0x55b2a73330dc <unknown>
#2 0x55b2a73387e9 <unknown>
#3 0x55b2a733a595 <unknown>
#4 0x55b2a733a61c <unknown>
#5 0x55b2a7379ae2 <unknown>
#6 0x55b2a73a2872 <unknown>
#7 0x55b2a7373d89 <unknown>
#8 0x55b2a73a2a3e <unknown>
#9 0x55b2a73c15c3 <unknown>
#10 0x55b2a73a2613 <unknown>
#11 0x55b2a73724f7 <unknown>
#12 0x55b2a7372e4e <unknown>
#13 0x55b2a761687b <unknown>
#14 0x55b2a761a921 <unknown>
#15 0x55b2a760236e <unknown>
#16 0x55b2a761b482 <unknown>
#17 0x55b2a75e6ccf <unknown>
#18 0x55b2a76400a8 <unknown>
#19 0x55b2a7640280 <unknown>
#20 0x55b2a764f7dc <unknown>
#21 0x7f2bc7b7b134 <unknown>


[8][pyyszhogzu] Task[0,1,로그인 실패]
[8][pyyszhogzu] 마지막 메모리 사용량: 13.09 MB
